dev-docs/publisher-api-reference.md

```diff
@@ -1290,4 +1285,79 @@ This call returns the following DFP video ad tag URL:
+
+{: .table .table-bordered .table-striped }
+| Field             | Type   | Description                                                                                                                                                                                                                                   | Example                                         |
+|-------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------|
+| iu                | string | (Required) DFP adUnit ID.  For more information, see the [DFP documentation on `iu`](https://support.google.com/dfp_premium/answer/1068325?hl=en#iu)                                                                                          | `/19968336/prebid_cache_video_adunit`           |
+| cust_params       | object | (Optional) Key-value pairs merged with Prebid's targeting values and sent to DFP on the video ad tag URL.  For more information, see the [DFP docs on `cust_params`](https://support.google.com/dfp_premium/answer/1068325?hl=en#cust_params) | `{section: "blog", anotherKey: "anotherValue"}` |
+| `description_url` | string | (Optional) Describes the video.  Required for Ad Exchange.  Prebid.js will build this for you unless you pass it explicitly.                                                                                                                  |                                                 |
+
+{: .alert.alert-info :}
+Note: Prebid.js will choose reasonable default values for any required DFP URL parameters that are not included in the `options.params` object.
+
+For more information about the options supported by the DFP API, see [the DFP API docs](https://support.google.com/dfp_premium/answer/1068325?hl=en#env).
+
+<a name="buildVideoUrl-usage-scenarios-and-expected-behavior" />
+
+#### Usage scenarios and expected behavior
+
+How you call `dfp.buildVideoUrl` may change depending on:
+
++ Whether you pass in query string parameters via `options.params`, the ad server URL via `options.url`, or both.
++ Whether you choose to use Prebid Cache, a server-side asset cache hosted by Prebid.org.
+
+Reasons you might pass the `options.url` field include:
+
+- You are not using an ad server, and you want to insert the winning VAST content into your player directly
+- You are using a video ad server that can accept a standalone VAST URL, as opposed to having to pass the VAST URL as a query string parameter  
+
+The table below lists the expected behavior of this method when called with various combinations of arguments and cache usage.
+
+{: .table .table-bordered .table-striped }
```

Comment:
I think we need to figure out how to more more clearly represent the information in the table below, as it seems a bit confusing to me at the moment.